test/test_car.py

```diff
@@ -0,0 +1,91 @@
+import pytest
+import time
+import numpy as np
+
+from vehicle_simulation_environment import car
+from scipy.integrate import quad
+
+
+TOLERANCE = 0.05
+
+
+class TestCar(object):
```

Comment:
```suggestion
class TestCar:
```
No need to inherit from `object`, this is implicitly done for you.